src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Create;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand\TableColumn;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand\TableColumn\TeradataTableColumnMeta;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Shared\Driver\MetaHelper;
+use Keboola\StorageDriver\Teradata\ConnectionFactory;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Column\ColumnCollection;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\TableBackendUtils\Table\Teradata\TeradataTableQueryBuilder;
+
+final class CreateTableHandler implements DriverCommandHandlerInterface
+{
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof CreateTableCommand);
+
+        // validate
+        assert($command->getPath()->count() > 0, 'CreateTableCommand.path is required');
```

Comment:
vynutil bych, že `path = 1` nic jiného do teradaty nedojde a když jo je to chyba. Pak tu path nemusíš iterovat.

Reply:
ok, hotovo